<ui version="4.0" >
 <class>lifetimeMainWidget</class>
 <widget class="QWidget" name="lifetimeMainWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="leftMargin" >
    <number>3</number>
   </property>
   <property name="topMargin" >
    <number>3</number>
   </property>
   <property name="rightMargin" >
    <number>3</number>
   </property>
   <property name="bottomMargin" >
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_state" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_state" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Start or stop horology</string>
        </property>
        <property name="text" >
         <string> Coming</string>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/kommt.xpm</iconset>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pause" >
        <property name="toolTip" >
         <string>want to make a break?</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/player_pause.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_time" >
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text" >
         <string>00:00:00</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin" >
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_info" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>3</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_state_info" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip" >
         <string>Current state</string>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text" >
         <string>idle</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin" >
         <number>3</number>
        </property>
        <property name="buddy" >
         <cstring>pushButton_info</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_info" >
        <property name="toolTip" >
         <string>Overview</string>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="application.qrc" >:/img/history.png</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_overview" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text" >
      <string>Start today:
End today:</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_state</tabstop>
  <tabstop>pushButton_pause</tabstop>
  <tabstop>pushButton_info</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
